ElrondNetwork/elrond-go
# En 7295 get username route

Added an API endpoint for getting the username for a given address.

Testing steps:
a) Prerequisites
-a testnet which has the following routes enabled in `api.toml`:
    - `address/:address/username`
    - `address/:address`
b) Test steps
-make API requests to the routes
c) Expected results
-on the `address/:address/username` route only the username should be returned
-the username should be visible also on the route `address/:address`

-if the given address exists and it has an username set, it should be returned.
-if the given address does not exist an error will be returned.
-if the given address exists, but it doesn't have an username set, an empty username will be returned.

d) Actual results
-tested on a local testnet and the acceptance criteria is met
